be/src/exec/workgroup/work_group.cpp

```diff
@@ -59,16 +101,17 @@ WorkGroupQueue& WorkGroupManager::get_io_queue() {
     return _wg_io_queue;
 }
 
-namespace {
-class DefaultWorkGroupInitialization {
-public:
-    DefaultWorkGroupInitialization() {
-        auto default_wg = std::make_shared<WorkGroup>("default_wg", 1, 10, 10, 10, WorkGroupType::WG_DEFAULT);
-        default_wg->init();
-        WorkGroupManager::instance()->add_workgroup(default_wg);
-    }
-} _;
-} // namespace
+DefaultWorkGroupInitialization::DefaultWorkGroupInitialization() {
+    auto default_wg = std::make_shared<WorkGroup>("default_wg", 0, 1, 20L * (1L << 32), 10, WorkGroupType::WG_DEFAULT);
```

Comment:
`1L << 32` is 4GB, maybe it should be `1L << 30`?

Reply:
ok